### DISCUSSION TITLE
How to get commands respect aliases

### DISCUSSION THREAD

**Discussion by nikero41** [knqyf263/pet#315] [Q&A] [answered]
I have an alias for eza at ls.

When I create a snippet, for example a simple `ls ~`, the response from `pet exec` is not calling eza but instead the builtin ls command.

Is there a way to make it respect my zshrc file?

**Comment by RamiAwar** [comment, marked as answer]
My first guess is that Pet is not using zsh, and hence not loading this alias of yours.

You can modify the General cmd configuration to make it use zsh. Try that and let me know if it works!
